src/lib/handle-result.ts

```diff
@@ -26,11 +35,15 @@ export function handleResult(entry: EntryVariants, functionMap: FunctionMap, sec
         case Variant.EndOfRecord:
             // we've left the test module, to prevent name conflicts clear the current FunctionMap
```

Comment:
It would be good to change this comment as well, either to mention the new `Map` as well, or (preferably) to remove the explicit mention and just describe why these maps need to be cleared.